devtools-ci-index.yaml

```diff
@@ -1586,6 +1586,67 @@
         - registry_devshift_credentials
     <<: *job_template_build_defaults
 
+- job-template:
+    name: '{ci_project}-planner-api-test-{test_url}'
+    triggers:
+      - timed: '0 0 * * * *'
+    wrappers:
+      - credentials-binding:
+        - text:
+            credential-id: 1b1d24cc-7bf5-46dc-a112-657de8f1388c
+            EE_TEST_OSIO_TOKEN: Token
+    scm:
+      - git:
+            url: https://github.com/{git_organization}/{git_repo}.git
+            shallow_clone: true
+            branches:
+                - master
+    builders:
+        - shell: |
+            set -x
+            export CICO_API_KEY=$(cat ~/duffy.key )
+
+            # get node
+            n=1
+            while true
+            do
+                cico_output=$(cico node get -f value -c ip_address -c comment)
+                if [ $? -eq 0 ]; then
+                    read CICO_hostname CICO_ssid <<< $cico_output
+                    if  [ ! -z "$CICO_hostname" ]; then
+                        # we got hostname from cico
+                        break
+                    fi
+                    echo "'cico node get' succeed, but can't get hostname from output"
+                fi
+                if [ $n -gt 5 ]; then
+                    # give up after 5 tries
+                    echo "giving up on 'cico node get'"
+                    exit 1
+                fi
+                echo "'cico node get' failed, trying again in 60s ($n/5)"
+                n=$[$n+1]
+                sleep 60
+            done
+            echo 'Using Host' $CICO_hostname
+
+            env > jenkins-env
+            set -e
+
+            # Run E2E tests
+            sshopts="-t -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -l root"
+            ssh_cmd="ssh $sshopts $CICO_hostname"
+            $ssh_cmd yum -y install rsync
+            rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload-tests
+            $ssh_cmd -t "cd payload-tests/EE_API_automation && /bin/bash cico_run_EE_tests.sh 
```

Comment:
I don't see a directory called `payload-tests` here https://github.com/fabric8io/fabric8-test

Reply:
And I don't see `cico_run_EE_tests.sh` in `EE_API_automation`:)

Reply:
@aditya-konarde payload-tests is a directory in your environment. See this line:
 rsync -e "ssh $sshopts" -Ha $(pwd)/ $CICO_hostname:payload-tests

@vpavlin i've sent a PR in https://github.com/fabric8io/fabric8-test to include cico_run_EE_tests.sh but it is not yet merged